### ISSUE TITLE
Log History issue : Nagios: ERROR: The request contains an invalid header. - Filter: Unexpected escape character.. (xx.xx.xx.xx:6557) 

### ISSUE THREAD

**Issue by GuillaumeSigma** [sni/Thruk#894]
**Describe the bug**
Trying to display history on host/service, I have the following message under Thruk (web interface) : 

```


No Backend available
None of the configured Backends could be reached, please have a look at the logfile for detailed information and make sure the core is up and running.


Nagios_XI: ERROR: The request contains an invalid header. - **Filter: Unexpected escape character.**. (172.31.210.52:6557)

```

The event log function in left pane works fine, probaly because it does not apply any filter on host/service.
The backend issue appears because livestatus stop the response that is too big (>100m)

**Thruk Version**
thruk 2.24-2
livestatus 1.5.0-p7
CentOS Linux release 7.5.1804 (Core)
Nagios Core 4.2.4

**To Reproduce**
1/ Open a page on any Service
2/ Clik View Alert History For This Service (Logs)

**Expected behavior**
Log for this service should appear.

**Screenshots**
[capture](https://user-images.githubusercontent.com/45846986/49935909-a59e4380-fed2-11e8-9399-67371ec5319b.JPG)

**Desktop (please complete the following information):**
Firefox 61.0.2 (64 bits)
Windows 10. (bug present with ie)

**Additional context**
Error in livestatus.log : 
```
2018-12-13 11:26:23 [client 19] request: GET log\n{16 lines follow...}
2018-12-13 11:26:23 [client 19] error: Filter: Unexpected escape character.
2018-12-13 11:26:23 [client 19] error: And: error combining filters for table 'log': expected 4 filters, but only 3 are on stack
2018-12-13 11:26:26 [client 19] Maximum response size of 104857600 bytes exceeded!

```

**Comment by sni** [comment]
Or the response is that big because of the filter error message. Is this reproducable somehow?

**Comment by kastesh** [comment]
Hello,
We have a similar error for the following bundle:

Thruk 2.26-2
nagios 4.1.1
livestatus 1.5.0p11


Error in thruk-logs:

```
[2019/02/05 14:49:45][nagios-server][DEBUG][thruk.log] > $VAR1 = 'GET log
Columns: class time type state host_name service_description plugin_output message options state_type contact_name
Filter: host_name = HOSTNAME
Filter: service_description = SERVICENAME
Filter: time >= 1549314000
Filter: time <= 1549400400
Filter: type = SERVICE ALERT
Filter: type = HOST ALERT
Filter: type = SERVICE FLAPPING ALERT
Filter: type = HOST FLAPPING ALERT
Filter: type = SERVICE DOWNTIME ALERT
Filter: type = HOST DOWNTIME ALERT
Filter: message ~ starting\\.\\.\\.
Filter: message ~ shutting\\ down\\.\\.\\.
Or: 8
And: 5
OutputFormat: json
ResponseHeader: fixed16
';
[2019/02/05 14:49:45][nagios-server][DEBUG][thruk.log] using new connection
[2019/02/05 14:49:46][nagios-server][DEBUG][thruk.log] header: 400          37
[2019/02/05 14:49:46][nagios-server][DEBUG][thruk.log] query status 400
[2019/02/05 14:49:46][nagios-server][ERROR][thruk.log] Error in: /thruk/cgi-bin/history.cgi
[2019/02/05 14:49:46][nagios-server][ERROR][thruk.log] The request contains an invalid header. - Filter: Unexpected escape character.
 at /usr/share/thruk/lib/Monitoring/Livestatus/Class/Lite.pm line 386.
 at /usr/lib64/thruk/perl5/Plack/Util.pm line 145.
    eval {...} called at /usr/lib64/thruk/perl5/Plack/Util.pm line 145
    Plack::Util::run_app('CODE(0x3063800)', 'HASH(0x4bcdd80)') called at /usr/lib64/thruk/perl5/Plack/Handler/FCGI.pm line 145
    Plack::Handler::FCGI::run('Plack::Handler::FCGI=HASH(0x2415c48)', 'CODE(0x3063800)') called at /usr/share/thruk/script/thruk_fastcgi.pl line 24
[2019/02/05 14:49:46][nagios-server][ERROR][thruk.log] ***************************
[2019/02/05 14:49:46][nagios-server][ERROR][thruk.log] No Backend available
[2019/02/05 14:49:46][nagios-server][ERROR][thruk.log] local livestatus: ERROR: The request contains an invalid header. - Filter: Unexpected escape character.. (/usr/local/nagios/var/rw/live)
[2019/02/05 14:49:46][nagios-server][ERROR][thruk.log] on page: http://nagios-server/thruk/cgi-bin/history.cgi?host=HOSTNAME&service=SERVICENAME&_=1549367384490
[2019/02/05 14:49:46][nagios-server][ERROR][thruk.log] User: nagiosadmin
[2019/02/05 14:49:46][nagios-server][INFO][thruk.log]  6160 Req: 013   mem:  43.64 MB   0.00 MB   dur: 0.864s  (0.853s/0.005s)   size:   13.689 kb   stat: 500   url: history.cgi?host=HOSTNAME&service=SERVICENAME&_=1549367384490

```
At livestatus log we see this two errors:
```
2019-02-05 14:53:33 [client 7] error: Filter: Unexpected escape character.
2019-02-05 14:53:33 [client 7] error: And: error combining filters for table 'log': expected 5 filters, but only 4 are on stack
```

**Comment by kastesh** [comment]
We use [logcache](https://www.thruk.org/documentation/logfile-cache.html) as workaround for this issue.
Perhaps this information will be useful to those who have the same error.

**Comment by sni** [comment]
sounds like this got fixed with #917
